ironfish/src/rpc/routes/fees/getFees.test.ts

```diff
@@ -1,55 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-
-import { useBlockWithTx, useMinerBlockFixture } from '../../../testUtilities'
-import { createRouteTest } from '../../../testUtilities/routeTest'
-import { GetFeesResponse } from './getFees'
-
-describe('Route chain.getFees', () => {
-  const routeTest = createRouteTest()
-
-  it('should fail if no block is found', async () => {
-    await expect(
-      routeTest.client.request('fees/getFees', { numOfBlocks: 2 }).waitForEnd(),
-    ).rejects.toThrow('numOfBlocks must be less than the current head sequence')
-  })
-
-  it('ignores miners fee from calculations', async () => {
-    const node = routeTest.node
-
-    const block = await useMinerBlockFixture(node.chain)
```

Comment:
we also need to delete the fixture file too

Reply:
removed